themes/courses.labster.com/lms/templates/ccx/enrollment.html

```diff
@@ -88,17 +89,15 @@ <h2> ${_("Student List Management")}</h2>
           <tbody>
             %for member in ccx_members:
             <tr>
-              <td>${member.user}</td>
```

Comment:
@lukyfebrianto why has it been removed? Does it affect somehow enrollment?

Reply:
@polesye on the ticket it said `when I open CCX Coach tab I cannot enroll students by username
And I don't see any information about this on UI.` that's why i remove that column.

No